x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/advanced_step/advanced_step_form.tsx

```diff
@@ -38,18 +40,33 @@ export const AdvancedStepForm: FC<CreateAnalyticsStepProps> = ({
   state,
   setCurrentStep,
 }) => {
+  const [advancedParamErrors, setAdvancedParamErrors] = useState<
```

Comment:
Nit: You could move the type spec to its own type `interface AdvancedParamErrors { ... }`.

Reply:
Set the advanced error type and used it in relevant files in 8837c39c1d075444f078c6d82434a820b8e2dbe3